modularizar.py

```diff
@@ -0,0 +1,62 @@
+import os
+import datetime #informa ano atual - ATP ETAPA2
+
+#mostrar e somente mostrar o nome ao usuario 
+
+print("Seja bem-vindo a loja de coelhos, entre e escolha um Rodiney Branta")
+
+Cargo_Atual = str(input("Para fazermos sua analise de credito, precisamos que voce entre com algumas informacoes:\nCargo atual:"))
+Salario_Atual = float(input("Salario Atual:"))
+Ano_Nascimento = int(input("Ano de nascimento:"))
+Ano_Atual = datetime.datetime.today() #ATP_ETAPA2
+Idade_Aproximada = (Ano_Atual.year - Ano_Nascimento) #ATP_ETAPA2
+Credito_Atual = ((Salario_Atual*(Idade_Aproximada / 1000)) +100) #ATP_ETAPA2
+meuNomeCompleto = "Rodiney Branta"
+
+os.system("cls")
```

Comment:
gambs... ✂️

Reply:
https://pypi.org/project/clear-screen/

Reply:
ou melhor ainda.. entender O QUE É LIMPAR A TELA: https://www.quora.com/Is-there-a-Clear-screen-function-in-Python/answer/Jim-Dennis-1

Reply:
> https://pypi.org/project/clear-screen/

[Commit](https://github.com/BrantaRodiney/CompThinking/pull/1/commits/b77fc925b28aec59d64fc2093cd049aafd17afc3) efetuado para correção através de uma função que limpa a tela, escrita por mim, já que o Python, não possui uma função built-in para tratar este problema.
O terminal que utilizo aceita character escaping através de um _print("\033[1;1f")_ – [ansi-escape sequence vt-100](http://ascii-table.com/ansi-escape-sequences-vt-100.php), entretanto observei que devido ao caracteres especiais, há casos, que o usuário pode utilizar deste recurso para ver algo que não deveria, e o programa acha que esta  executando outras coisas.